Content.Shared/Body/Systems/SharedBodySystem.Body.cs

```diff
@@ -308,9 +308,9 @@ private void SetupOrgans(Entity<BodyPartComponent> ent, Dictionary<string, strin
         foreach (var part in parts)
         {
 
-            _gibbingSystem.TryGibEntityWithRef(bodyId, part.Id, GibType.Gib, GibContentsOption.Skip, ref gibs,
```

Comment:
This is by design, detaching all parts and dropping them looks really bad. Additionally, the off center nature of bodypart sprites means that impulse based scattering looks really really bad.

Reply:
Gibbing is also only used in cases where a body should be completely destroyed such as receiving an excessive amount of damage or explosions. In both these cases, bodyparts would not get dropped at full health. 

Instead of dropping bodyparts, you should add giblet items (basically chunks of meat) that get spawned when gibbing. (These are defined on the gibbable component)